src/main/java/internity/core/InternshipList.java

```diff
@@ -1,11 +1,10 @@
 package internity.core;
 
-import internity.ui.Ui;
-
 import java.util.ArrayList;
-
 import java.util.logging.Logger;
 
+import internity.ui.Ui;
+
 public class InternshipList {
     public static final int INDEX_MAXLEN= 5;
```

Comment:
I know this isn't your line of code but could you ammend this.

```suggestion
    public static final int INDEX_MAXLEN = 5;
```